packages/database/src/DatabaseType.js

```diff
@@ -134,6 +134,19 @@ export class DatabaseType {
   async save() {
     const data = await this.getData();
     if (this.id) {
+      // if any columns used custom selectors, updating is not supported via save()
```

Comment:
This is to handle a case like
```
const entity = models.entity.findOne({ code: 'TO' });
entity.name = 'Tonga is cool';
entity.save(); // would throw an error before, because entity.region and entity.bounds were extracted into a readable format, but saving them back to the db in that format isn't supported
```